src/core/utils/toast/useToast.ts

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+
+import { IToastContextState, ToastContext } from './ToastContext';
+
+export const useToast = (): IToastContextState => {
```

Comment:
Burde denne være 'export default'?

Reply:
I generally prefer not using default exports.
This enforces the name of the exported function and generally makes things easier to import automatically with with the editor.